NOTICE.adoc

```diff
@@ -0,0 +1,21 @@
+= Notices for the MuleSoft Documentation UI Project
+
+This UI for Antora is produced and maintained by the MuleSoft Documentation team.
+
+== Copyright
+
+This UI is derived from the https://gitlab.com/antora/antora-ui-default/[Antora Default UI].
+Modifications have been applied to align the theme and behavior of the UI with the MuleSoft brand and documentation requirements.
+All branding elements in this UI, including icons, images, and fonts, are copyright Salesforce.
+
+== Declared Project Licenses
```

Comment:
I'd move this section below the section Source Code, nest it (`===`), and change the title to "Source Code Licenses"